### DISCUSSION TITLE
Yugabyte Datastore for Multi Region Support

### DISCUSSION THREAD

**Discussion by sameer-m-dev** [openfga/community#243] [Ideas]
### Checklist

- [X] I agree to the terms within the [OpenFGA Code of Conduct](https://github.com/openfga/.github/blob/main/CODE_OF_CONDUCT.md).

### Describe the problem you'd like to have solved

Hi, I am currently trying to understand the support OpenFGA has for distributed databases such as Yugabyte or CockroachDB. We are trying to build a global auth system that requires a fast and reliable data backend for storage and data access. While I have gone through the documentation of OpenFGA, I could not find a source where multi-region support has been talked about. Clarification on the same would be much appreciated

### Describe the ideal solution

Use a distributed database such as Yugabyte or CockroachDB as the backend for OpenFGA

**Comment by aaguiarz** [comment]
@sameer-m-dev we plan to add support for Dynamo DB soon https://github.com/openfga/roadmap/issues/49. We don't have current plans for other databases yet.
